app/controllers/donations_controller.rb

```diff
@@ -11,6 +11,15 @@ def create
     return_to_origin_page
   end
 
+  def undo
+    params[:donation] = {
+                         "candidate_issue_id" => session[:undo].keys.first,
+                                     "amount" => session[:undo].values.first
+                        }
```

Comment:
Indent the right brace the same as the start of the line where the left brace is.